app/controllers/pages_controller.rb

```diff
@@ -1,4 +1,5 @@
 class PagesController < ApplicationController
   def home
+    render layout: 'wide'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.